package.json

```diff
@@ -62,5 +63,8 @@
     "ts",
     "type",
     "types"
-  ]
+  ],
+  "dependencies": {
+    "scrollparent": "^2.0.1"
```

Comment:
i'm not a big fan of adding a library like this, we can internalize the code so we know what's inside

Reply:
scrollparent https://www.npmjs.com/package/scrollparent is very widely used and stable (363,995 Weekly Downloads) I don't see the benefit of copying scrollparent's code to this repository

Reply:
I want the library to be self-contained